### ISSUE TITLE
Implement option to keep active partition active if all requirements are fulfilled

### ISSUE THREAD

**Issue by jfriesse** [corosync/corosync-qdevice#7]
Idea is to add option which, if activated, would try to keep vote for current active partition if all other requirements are fulfilled.

Idea based on report from ML:
```
Hi ,

I am planning to use Corosync Qdevice version 3.0.0 with corosync version 2.4.4 and pacemaker 1.1.16 in a two node cluster.

I want to know if failback can be avoided in the below situation.


  1.  The pcs cluster is in split brain scenario after a network break between two nodes. But both nodes are visible and reachable from qdevice node.
  2.  The qdevice with ffsplit algorithm selects node with id 1 (lowest node id) and node 1 becomes quorate.
  3.  Now if node 1 goes down/is not reachable from qdevice node , the node 2 becomes quorate.

But when node 1 becomes again reachable from qdevice , it becomes quorate and node 2 again goes down. i.e The resources failback to node 1.

Is there any way to prevent this failback.


With Regards
Somanath Thilak J
```

**Comment by jfriesse** [comment]
This is needed also to make 2 node cluster work correctly with knet where restarted node may get quorum because creation of single node membership first. Relevant ML discussion https://lists.clusterlabs.org/pipermail/users/2020-April/027087.html

**Comment by jfriesse** [comment]
This issue is now fixed via fbc34f3b05ebecef59b26ac8ee6e1e0dc7c779e9, 09c6f788640cd3bcfe074511dd32dd5883d0fe66 and  7a0201a5c6081f306565082917c3b7920b01b4da so closing for now.
